### PR TITLE
py-sonata-network-reduction v0.0.8

### PR DESCRIPTION
**Pull request by asanin-epfl**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-neuron-reduce/package.py [hunk 1]
```diff
@@ -14,8 +14,9 @@ class PyNeuronReduce(PythonPackage):
     git = "git@github.com:orena1/neuron_reduce.git"
 
     version('develop', branch='master')
-    version('0.0.6', sha256='e75ce138ae049f550d72ec86c5cae4afcada577a2805d9dc50cbe80eff1f2256')
+    version('0.0.8', commit='3aada2ad3606723a6ebbaf39581153e38de24733')
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-neuron-reduce/package.py:17]: Tags?

**Review by matz-e — DISMISSED** [review]

File: var/spack/repos/builtin/packages/py-sonata-network-reduction/package.py [hunk 2]
```diff
@@ -13,26 +13,27 @@ class PySonataNetworkReduction(PythonPackage):
...
     depends_on('py-joblib@0.14:', type='run')
     depends_on('py-bluepyopt@1.8.68:', type='run')
     depends_on('py-bglibpy@4.2:', type='run')
-    depends_on('py-morphio@2.3.4:', type='run')
+    depends_on('py-morphio@2.3.9:', type='run')
     depends_on('neuron+python', type='run')
     depends_on('py-aibs-circuit-converter@0.0.3:', type='run')
-    depends_on('py-bluepysnap@0.1.2:', type='run')
-    depends_on('py-neuron-reduce@0.0.7:', type='run')
+    depends_on('py-bluepysnap@0.3.0:', type='run')
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-sonata-network-reduction/package.py]: Come to think of it, is this version released?

Comment (asanin-epfl) [reply var/spack/repos/builtin/packages/py-sonata-network-reduction/package.py]: fixed

File: var/spack/repos/builtin/packages/py-neuron-reduce/package.py (hunk 1 above)
Comment (asanin-epfl) [reply var/spack/repos/builtin/packages/py-neuron-reduce/package.py:17]: Unfortunately the author of the package didn't release. I have to use commits.

Comment (matz-e) [reply var/spack/repos/builtin/packages/py-neuron-reduce/package.py:17]: OK

**Comment by asanin-epfl** [comment]
Retest this please

**Review by matz-e — DISMISSED** [review]
I'll see about Travis

**Comment by tomdele** [comment]
don t you have to add it to the bbp-packages ?

**Comment by asanin-epfl** [comment]
Thank you! I completely forgot about it.

**Review by matz-e — APPROVED** [review]
Forgot about packages, too, sorry!

**Review by tomdele — APPROVED** [review]

**Pull request merged**